package.json

```diff
@@ -5,14 +5,16 @@
   "author": "Ephox Inc",
   "license": "Apache-2.0",
   "scripts": {
+    "package-name": "echo $npm_package_name",
```

Comment:
Do you still need these?

Reply:
Completely forgot about those. I mean they dont hurt anybody. But I will remove them nonetheless